apps/codeforafrica/src/payload/blocks/GetInTouch.js

```diff
@@ -0,0 +1,29 @@
+import link from "../fields/links/link";
+
+const GetInTouch = {
+  slug: "get-in-touch",
+  imageURL: "/images/cms/blocks/get_in_touch.jpg",
+  imageAltText: "Display Get In Touch Call to Action",
+  fields: [
+    {
+      name: "title",
+      label: "Title",
+      required: true,
+      type: "text",
+    },
+    {
+      name: "subtitle",
+      label: "Subtitle",
+      required: true,
+      type: "text",
+    },
+    {
+      name: "action",
+      label: "Action",
+      type: "group",
+      fields: [link()],
+    },
```

Comment:
What structure do we get here (JSON) @kelvinkipruto ?

Reply:
@kilemensi The JSON Response looks like this:
```
{
  "action": {
    "label": "GET IN TOUCH",
    "linkType": "internal",
    "doc": {...},
    "href": "/contact"
  }
}
```